### PR TITLE
✨AMP-Consent: Add Marfeel to CMPs list

### PR DESCRIPTION
**Pull request by pedrolopezmrf**
- Add Marfeel to CMPs list with its proper parameters.
- Added documentation:
    - Instructions
    - Example
    - Example on cmp-vendors page.

I2i opened issue can be found [here](https://github.com/ampproject/amphtml/issues/26181).

/cc @ampproject/wg-monetization

### CODE REVIEW

File: examples/amp-consent/cmp-vendors.amp.html [hunk 1]
```diff
@@ -164,6 +165,24 @@ <h3>Image that is NOT blocked by consent</h3>
...
         <!-- End didomi example -->
 
+      <!-- Marfeel example -->
+      <amp-consent id='marfeelConsent' layout='nodisplay' type='Marfeel'>
+        <script type="application/json">
+          {
+            "postPromptUI": "postPromptUI",
+            "clientConfig": {
+              "consentLanguage": "en",
+              "publisherLogo": "https://marfeel.mgr.consensu.org/amp/img/marfeel-logo.png",
```
Comment (zhouyx) [inline examples/amp-consent/cmp-vendors.amp.html]: nit, please remove the trailing `,` so it's a valid json object. Thanks

Comment (oliverfernandez) [reply examples/amp-consent/cmp-vendors.amp.html]: Fixed

**Comment by oliverfernandez** [comment]
@zhouyx I commited the requested change because @pedrolopezmrf is on holidays. I also realized we had a typo in the prompt button

File: extensions/amp-consent/cmps/marfeel.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
+<!---
+Copyright 2019 The AMP HTML Authors. All Rights Reserved.
```
[the remark this answers is not in the document]
Comment (oliverfernandez) [reply extensions/amp-consent/cmps/marfeel.md]: Travis build failed with the license update :S

**Comment by zhouyx** [comment]
Looks great. Merged! Thank you for the integration.

**Pull request merged**